### PR TITLE
Adds transak for testing on ASTR and SDN

### PR DESCRIPTION
**Pull request by leohar**
[Network naming](https://www.notion.so/d20db13dcacb4ec8801e8869e2e5975e?v=9bcdb10756834041811b64d527b5fdc1)

### CODE REVIEW

File: chains/v4/chains_dev.json [hunk 1]
```diff
@@ -1106,7 +1106,12 @@
                 "assetId": 0,
                 "symbol": "SDN",
                 "precision": 18,
-                "priceId": "shiden"
+                "priceId": "shiden",
+                "buyProviders": {
+                     "transak": {
+                        "network": "SHIDEN"
+                    }
```
Comment (stepanLav) [inline chains/v4/chains_dev.json:1113]:
```suggestion
                     }
```

File: chains/v4/chains_dev.json [hunk 2]
```diff
@@ -2461,7 +2466,12 @@
                 "assetId": 0,
                 "symbol": "ASTR",
                 "priceId": "astar",
-                "precision": 18
+                "precision": 18,
+                "buyProviders": {
+                     "transak": {
+                        "network": "ASTAR"
+                    }
```
Comment (stepanLav) [inline chains/v4/chains_dev.json:2473]:
```suggestion
                     }
```

File: chains/v5/chains_dev.json [hunk 3]
```diff
@@ -1106,7 +1106,12 @@
                 "symbol": "SDN",
                 "precision": 18,
                 "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/chains/white/Shiden.svg",
-                "priceId": "shiden"
+                "priceId": "shiden",
+                "buyProviders": {
+                     "transak": {
+                        "network": "SHIDEN"
+                    }
```
Comment (stepanLav) [inline chains/v5/chains_dev.json:1113]:
```suggestion
                     }
```

File: chains/v5/chains_dev.json [hunk 4]
```diff
@@ -2458,7 +2463,12 @@
                 "symbol": "ASTR",
                 "priceId": "astar",
                 "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/chains/white/Astar.svg",
-                "precision": 18
+                "precision": 18,
+                "buyProviders": {
+                     "transak": {
+                        "network": "ASTAR"
+                    }
```
Comment (stepanLav) [inline chains/v5/chains_dev.json:2470]:
```suggestion
                     }
```

File: chains/v4/chains_dev.json [hunk 5]
```diff
@@ -1106,7 +1106,12 @@
                 "assetId": 0,
                 "symbol": "SDN",
                 "precision": 18,
-                "priceId": "shiden"
+                "priceId": "shiden",
+                "buyProviders": {
+                     "transak": {
```
Comment (stepanLav) [inline chains/v4/chains_dev.json:1111]:
```suggestion
                    "transak": {
```

Comment (leohar) [reply chains/v4/chains_dev.json]: fixed

**Pull request merged**